src/libs/Navigation/AppNavigator/PublicScreens.tsx

```diff
@@ -67,6 +68,11 @@ function PublicScreens() {
                 component={PublicRightModalNavigator}
                 options={rootNavigatorScreenOptions.rightModalNavigator}
             />
+            <RootStack.Screen
+                name={NAVIGATORS.TEST_TOOLS_MODAL_NAVIGATOR}
+                options={rootNavigatorScreenOptions.basicModalNavigator}
```

Comment:
We can create a custom screen options here with the native option for the native to enable the overlay:

```suggestion
                options={{
                    ...rootNavigatorScreenOptions.basicModalNavigator,
                   native: {
                      contentStyle: {
                          backgroundColor: theme.overlay
                      }
                  }
                }}
```
We can confirm the backdrop color with design team later:

https://github.com/user-attachments/assets/f796ecb8-3db3-409c-a25b-2d7254a6b1a5

Reply:
Now we have another thing to address. Click on overlay to dismiss the modal. You can see the references here:
https://reactnavigation.org/docs/stack-navigator/#transparent-modals

> If you want to further customize how the dialog animates, or want to close the screen when tapping the overlay etc., you can use the useCardAnimation hook to customize elements inside your screen.